docs/spelling_wordlist

```diff
@@ -17,12 +17,15 @@ cancelled
 clickable
 closable
 codepoint
+config
```

Comment:
If we're referring to `paths.config`, then it should be quoted; if it's "the config file" then the English text should be expanded to "configuration" as appropriate.